src/views/self/view_weapons.cpp

```diff
@@ -330,7 +334,7 @@ namespace big
 				for (auto& weapon_hash : g.weapons.weapon_hotkeys[selected_key])
 				{
 					ImGui::PushID(counter);
-					auto weapon = g_gta_data_service->weapon_by_hash(weapon_hash);
+					weapon_item weapon = g_gta_data_service->weapon_by_hash(weapon_hash);
```

Comment:
Any reason for changing the type from `auto` to `weapon_item`?

Reply:
nope, just don't like auto